src/Admin/AbstractAdmin.php

```diff
@@ -1690,10 +1695,12 @@ public function getPerPageOptions(): array
 
     /**
      * Returns true if the per page value is allowed, false otherwise.
+     *
+     * @param mixed $perPage
      */
-    final public function determinedPerPageValue(int $perPage): bool
+    final public function determinedPerPageValue($perPage): bool
```

Comment:
I think we should stick to `int` here, why `mixed`?

Reply:
The usage is
```
       if (!isset($parameters['_per_page']) || !$this->determinedPerPageValue($parameters['_per_page'])) {
            $parameters['_per_page'] = $this->getMaxPerPage();
        }
```

We can't be sure that `$parameters['_per_page']` is an int.
I can add `is_int` check here instead if you prefer

Reply:
Done